execute/plugin_functions.go

```diff
@@ -389,7 +389,7 @@ func mergeCommitObservations(
 	validators := make(map[cciptypes.ChainSelector]consensus.OracleMinObservation[exectypes.CommitData])
 	for selector, f := range fChain {
 		validators[selector] =
-			consensus.NewOracleMinObservation[exectypes.CommitData](consensus.FPlus1(f), nil)
+			consensus.NewOracleMinObservation[exectypes.CommitData](consensus.TwoFPlus1(f), nil)
```

Comment:
I raised this offline but this might be pretty suboptimal because we'd kick out the root in particular that is giving issues, and then we'd try to execute whatever's left, which may not work due to nonce issues.

We should properly address this in a follow-up.